### ISSUE TITLE
Table editor doesn't show when the table is inside another tag.

### ISSUE THREAD

**Issue by abram27** [froala/wysiwyg-editor#4155]
##### Expected behavior.
Editor popup should show when table is inside another tag:
![Editor](https://user-images.githubusercontent.com/17630795/106045446-8bba1d80-60a6-11eb-92ba-9f42a35b7060.png)

##### Actual behavior.
When add div tag around table, the popup to edit the table does not show.

##### Steps to reproduce the problem.
1. Go to https://froala.com/wysiwyg-editor/examples/resize-table/ 
2. In code view add div tag around table
3. In standard view click on table cell
4. Editor for table doesn't display

##### Editor version.
3.2.3

##### OS.
Windows 10 19042.746

##### Browser.
Chrome 88.0.4324.104

##### Recording.
https://user-images.githubusercontent.com/17630795/106045076-26fec300-60a6-11eb-83ac-d5505f2d4641.mp4

**Comment by ilyaskarim** [comment]
The issue is being scheduled for a fix in V3.6.

**Comment by earshinov** [comment]
The stack trace (from #4198):

```
Uncaught TypeError: Cannot read property 'length' of undefined
    at _currentSelection (table.js?version=1.0.1:1715)
    at _selectionOffset (table.js?version=1.0.1:1817)
    at _showEditPopup (table.js?version=1.0.1:92)
    at HTMLTableCellElement._mouseUp (table.js?version=1.0.1:2030)
    at froala_editor.pkgd.min.js?version=1.0.1:7
```

**Comment by kaniansky** [comment]
why would a new bug be fixed only in 4th upcoming minor version?

**Comment by funnelitadmin** [comment]
i am also facing same issue in angular 11

**Comment by HiHub-Online** [comment]
In v3.2.4 to table (in another tag) popup shows correct.
But for all versions after no shown.

**Comment by shnigi** [comment]
Have the same problem in 3.2.0 and 3.2.6

**Comment by earshinov** [comment]
I don't see anything related in the changelog for v3.2.7 (https://github.com/froala/wysiwyg-editor/releases/tag/v3.2.7). Still waiting...

**Comment by MeghashreeAP** [comment]
Thank you for your feedback.
This issue is not reproducible on v4.0.9 branch.

**Comment by MeghashreeAP** [comment]
Thank you for your feedback.
This issue is not reproducible on latest branch.

**Comment by earshinov** [comment]
I can confirm that this issue is fixed in 4.0.9

**Comment by ilyaskarim** [comment]
Issue is being fixed
